odkx-src/sync-endpoint-user-instructions.rst

```diff
@@ -7,25 +7,48 @@
   subdomain
 
 
+Users and Groups
+=================
 
+A user needs to be assigned to a group within ODK-X sync-endpoint to set their permissions and roles for ODK-X apps. More information about groups and the roles is available in :ref:`Data Permission Filters<data-permission-filters>` section.
```

Comment:
Sync Endpoint should be in caps and I don't think it should have a hyphen. No "the" before roles.